test/language/comments/hashbang-not-empty.js

```diff
@@ -0,0 +1,10 @@
+#! these characters should be treated as a comment
```

Comment:
I definitely need to change the linter to not fail for this. I believe it will say you're missing the copyright lines here...